source/Operation Guides/InterFlux Masternodes/Using Masternode Setup Tool/masternode-setup-tool.rst

```diff
@@ -0,0 +1,309 @@
+desktop. The desktop application should resemble the 
+following interface.
+
+.. image:: media/image11.png
+   :width: 6.26806in
+   :height: 3.27778in
+
+You now have two options: either run the STRAX Blockchain 
+and the Cirrus Sidechain via the Masternode dashboard, or 
+initiate Masternode registration within the Federation.
+
+
+Run Masternode
+==============
+If you are already a member of the Federation, simply 
+launch the app and click on the "Run Masternode" button. 
+The app will initialize and synchronize both the 
+STRAX Blockchain and the Cirrus Sidechain. Completion 
+time varies based on your system specifications. Upon 
+completion, the following screen will appear.
+
+.. image:: media/image12.png
+   :width: 6.26806in
+   :height: 3.27778in
+
+Setup Masternode
+================
+
+To join the Federation, launch the app and select the 
+"Setup Masternode" button. Follow the setup steps as prompted. 
+The app will initialize and synchronize both the STRAX 
+Blockchain and the Cirrus Sidechain. It will also guide you 
+through the setup process, requesting necessary information. 
+Completion time varies depending on your system specifications. 
+Upon completion, the following screen will appear.
+
+1. Agree to the End-User License Agreement.
+   
+.. image:: media/image13.png
+   :width: 6.26806in
+   :height: 3.27778in
+
+2. Create new or use existing Federation key.
+
+.. image:: media/image14.png
+   :width: 6.26806in
+   :height: 3.27778in
+
+3. Choose an option to create, restore, or use an existing wallet.
+
+.. image:: media/image15.png
+   :width: 6.26806in
+   :height: 3.27778in
+
+4. In this example, we will use an existing mining wallet 
+   from the `Wallet Configuration <#wallet-configuration>`__ 
+   section earlier in this guide. You will be prompted 
+   to select the wallet to where you have transferred your collateral.
+
+.. image:: media/image16.png
+   :width: 6.26806in
+   :height: 3.27778in
+
+5. Enter the password for your STRAX (collateral) wallet twice for confirmation.
+
+.. image:: media/image17.png
+   :width: 6.26806in
+   :height: 3.27778in
+
+6. You will be prompted to select the mining wallet to where you have 
+   transferred your mining fee.
+
+.. image:: media/image18.png
+   :width: 6.26806in
+   :height: 3.27778in
+
+7. Enter the password for your Cirrus (mining) wallet twice for confirmation.
+
+.. image:: media/image19.png
+   :width: 6.26806in
+   :height: 3.27778in
+
+8. You will be prompted to select the collateral address from the collateral 
+   wallet selected on previous steps.
+
+.. image:: media/image20.png
+   :width: 6.26806in
+   :height: 3.27778in
+
+9. Upon successful verification of collateral height and mining fee, 
+   the following screen will appear, and the Masternode Dashboard will launch.
+
+.. image:: media/image21.png
+   :width: 6.26806in
+   :height: 3.27778in
+
+Congratulations, you have successfully completed the registration and are now 
```

Comment:
Can we also highlight that you do not become an active member for 240 CRS Blocks (Est. 64mins) and that the dashboard will highlight mining status.